src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/NetworkAddressChange.OSX.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net.NetworkInformation
+{
+    // OSX implementation of NetworkChange
+    public class NetworkChange
+    {
+        static public event NetworkAddressChangedEventHandler NetworkAddressChanged
+        {
+            add
+            {
+                throw new NotImplementedException();
+            }
+            remove
+            {
+                throw new NotImplementedException();
```

Comment:
I know these can be implemented, but I won't have time to finish them before Wednesday (most likely). We need to use some OSX-specific functions which will take a bit of time to get familiar with. I'm mainly using "NotImplementedException" to mean "not yet implemented but I know how", and "PlatformNotSupportedException" to mean "I couldn't find a way to implement this at all"